README.md

```diff
@@ -52,21 +52,31 @@ $logProcessor->setLogInfo(new LogInfo(...));
 ```
 
 ## Development
-### AWS resources
-Use the provided `test-cf-stack.json` to create a CloudFormation stack. Use the outputs to set environment variables
-`AWS_DEFAULT_REGION`, `S3_LOGS_BUCKET`. Create an access key for the generated user. Set it to the environment 
-variables `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`.
 
-### Azure resources
-Use the provided `test-arm-template.json` to create ARM stack:
+Prerequisites:
+* locally installed `terraform` and `jq`
+    * https://www.terraform.io
+    * https://stedolan.github.io/jq
+* configured `az` and `aws` CLI tools (run `az login` and `aws configure --profile YOUR_AWS_PROFILE_NAME`)
 
-    az group create --name testing-api-error-control --location "East US"
+### AWS and Azure resources
 
-    az deployment group create --resource-group testing-api-error-control --template-file test-arm-template.json --parameters storage_account_name=testingapierrorcontrol container_name=test-container
+```shell
+export NAME_PREFIX= # your name/nickname to make your resource unique & recognizable
+export AWS_PROFILE=xxxx # profile name for AWS account of your team
+export AZURE_SUBSCRIPTION_ID=xxxx # subscription id of your team
```

Comment:
Tohle bych dal pryc a zadratoval to primo do TF. Proste `api-error-control` se vyviji pod PS dev accountem. Latest & greatest approach je asi tady
https://github.com/keboola/sandboxes-service/blob/8406d9dc004904dd374f2c6f3337f93945fe3358/provisioning/local/aws.tf#L2
https://github.com/keboola/sandboxes-service/blob/4f51346d62ab0121f092a618e0ddeab1ec2707b4/provisioning/local/azure.tf#L9

Reply:
Tohle je libka, navic hodne obecna a patrne se bude pak pouzivat i v Connection az zahodej Zend. Proto mi prislo zbytecny tam mit natvrdo profily/subskripce. U queue a tak bych to chapal, ale tady to nedava smysl.

Reply:
Zmeneno viz debata na slacku